cypress/integration/data-import/mrc-import-file-for-update-items.spec.js

```diff
@@ -213,11 +214,77 @@ describe('ui-data-import: MARC file upload with the update of instance, holding,
   const mappingProfileNameForExport = `autoTestMappingProf.${getRandomPostfix()}`;
   const jobProfileNameForExport = `autoTestJobProf.${getRandomPostfix()}`;
 
+  const collectionOfMappingAndActionProfiles = [
+    {
+      mappingProfile: { typeValue: NewMappingProfile.folioRecordTypeValue.instance,
+        name: mappingProfileNameForInstance,
+        fillProfile: NewMappingProfile.fillInstanceMappingProfile },
+      actionProfile: { typeValue: NewActionProfile.folioRecordTypeValue.instance,
+        name: actionProfileNameForInstance,
+        action: 'Update (all record types except Orders)' }
+    },
+    {
+      mappingProfile: { typeValue: NewMappingProfile.folioRecordTypeValue.holdings,
+        name: mappingProfileNameForHoldings,
+        fillProfile: NewMappingProfile.fillHoldingsMappingProfile },
+      actionProfile: { typeValue: NewActionProfile.folioRecordTypeValue.holdings,
+        name: actionProfileNameForHoldings,
+        action: 'Update (all record types except Orders)' }
+    },
+    {
+      mappingProfile: { typeValue : NewMappingProfile.folioRecordTypeValue.item,
+        name: mappingProfileNameForItem,
+        fillProfile: NewMappingProfile.fillItemMappingProfile },
+      actionProfile: { typeValue: NewActionProfile.folioRecordTypeValue.item,
+        name: actionProfileNameForItem,
+        action: 'Update (all record types except Orders)' }
+    }
+  ];
+
+  const collectionOfMatchProfiles = [
+    {
+      matchProfile: { profileName: matchProfileNameForInstance,
+        incomingRecordFields: {
+          field: '001'
+        },
+        existingRecordFields: {
+          field: '001'
+        },
+        matchCriterion: 'Exactly matches',
+        existingRecordType: 'MARC_BIBLIOGRAPHIC' }
+    },
+    {
+      matchProfile: { profileName: matchProfileNameForHoldings,
+        incomingRecordFields: {
+          field: '901',
+          subfield: 'a'
+        },
+        matchCriterion: 'Exactly matches',
+        existingRecordType: 'HOLDINGS',
+        holdingsOption: NewMatchProfile.optionsList.holdingsHrid }
+    },
+    {
+      matchProfile: {
+        profileName: matchProfileNameForItem,
+        incomingRecordFields: {
+          field: '902',
+          subfield: 'a'
+        },
+        matchCriterion: 'Exactly matches',
+        existingRecordType: 'ITEM',
+        itemOption: NewMatchProfile.optionsList.itemHrid
+      }
+    }
+  ];
+
+  const jobProfileForUpdate = {
+    ...NewJobProfile.defaultJobProfile,
+    profileName: jobProfileNameUpdate,
+    acceptedType: NewJobProfile.acceptedDataType.marc
+  };
+
   beforeEach(() => {
-    cy.login(
-      Cypress.env('diku_login'),
-      Cypress.env('diku_password')
-    );
+    cy.login(Cypress.env('diku_login'), Cypress.env('diku_password'));
```

Comment:
There is method cy.loginAsAdmin. Please use it

Reply:
Done